src/us/kbase/workspace/test/workspace/WorkspaceTester.java

```diff
@@ -1140,41 +1138,33 @@ protected void getNonExistantObject(WorkspaceUser foo, ObjectIdentifier oi,
 			new ResolvedWorkspaceID(1, "foo", false, false);
 	
 	protected void testObjectIdentifier(String goodId) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId);
 		new ObjectIDResolvedWS(RWSID, goodId);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
 		new ObjectIDNoWSNoVer(goodId);
 	}
 	
 	protected void testObjectIdentifier(String goodId, int version) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId, version);
 		new ObjectIDResolvedWS(RWSID, goodId, version);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
 		new ObjectIDNoWSNoVer(goodId);
 	}
 	
 	protected void testObjectIdentifier(int goodId) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId);
 		new ObjectIDResolvedWS(RWSID, goodId);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
 		new ObjectIDNoWSNoVer(goodId);
 	}
 	
 	protected void testObjectIdentifier(int goodId, int version) {
-		new ObjectIdentifier(new WorkspaceIdentifier("foo"), goodId, version);
 		new ObjectIDResolvedWS(RWSID, goodId, version);
 //		new ObjectIDResolvedWSNoVer(fakews, goodId);
```

Comment:
are these commented-out lines here for a reason?

Reply:
It looks like that class as been converted to be a package only class and so its methods aren't accessible. It's been years since I commented those out and I have no idea why they're there.

Anyway, I've removed them locally. The change will show up in 5 PRs.